packages/core/src/runtime/utils/trimSelector.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Trims selectors to generate consistent hashes.
+ */
+export function trimSelector(selector: string): string {
```

Comment:
for me it's a little bit more than a nit since what you're doing here isn't actually trimming. Youre're specifically targeting the spaces after the `>` combinator.

I'd rename this to `stripExtraWhitespace` and add some extra documentation why that specifically is necessary

Reply:
Potentially we could have more cases, that's why I went with more general naming.